Kernel/FileSystem/SysFS/Subsystems/Kernel/RequestPanic.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/FileSystem/SysFS/Subsystems/Kernel/RequestPanic.h>
+#include <Kernel/Library/Panic.h>
+#include <Kernel/Security/Jail.h>
+#include <Kernel/Tasks/Process.h>
+
+namespace Kernel {
+
+UNMAP_AFTER_INIT NonnullRefPtr<SysFSSystemRequestPanic> SysFSSystemRequestPanic::must_create(SysFSDirectory const& parent_directory)
+{
+    return adopt_ref_if_nonnull(new (nothrow) SysFSSystemRequestPanic(parent_directory)).release_nonnull();
+}
+
+static ErrorOr<void> check_current_process_not_jailed()
```

Comment:
Do we not have a helper for jails that already returns a bool for this condition?